src/extension.ts

```diff
@@ -280,16 +281,19 @@ async function handleError(error: unknown, topic: string, defaultMessage: string
         getLogger().verbose(`${topic}: user cancelled`)
         return
     }
-
     const logsItem = localize('AWS.generic.message.viewLogs', 'View Logs...')
     const logId = getLogger().error(`${topic}: %s`, error)
     const message = resolveErrorMessageToDisplay(error, defaultMessage)
 
-    await vscode.window.showErrorMessage(message, logsItem).then(async resp => {
-        if (resp === logsItem) {
-            await Logging.declared.viewLogsAtMessage.execute(logId)
-        }
-    })
+    if (error instanceof ToolkitError && error.documentationUri) {
+        await showMessageWithUrl(message, error.documentationUri, 'Go to Documentation', 'error')
```

Comment:
can use `viewDocs` https://github.com/aws/aws-toolkit-vscode/blob/909056f87a88a8757818615c81db7abbea834fa5/src/shared/localizedText.ts#L22

```suggestion
        await showMessageWithUrl(message, error.documentationUri, 'View Documentation', 'error')
```